packages/mechaml/mechaml.0.1/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+maintainer: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+authors: "Yann Hamdaoui <yann.hamdaoui@gmail.com>"
+homepage: "https://github.com/yannham/mechaml"
+bug-reports: "https://github.com/yannham/mechaml/issues"
+license: "LGPL v3"
+dev-repo: "git://github.com/yannham/mechaml.git"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+build-test: [
+  ["./configure" "--enable-tests"]
+  [make "test"]
+]
+build-doc: ["make doc"]
+remove: ["ocamlfind" "remove" "mechaml"]
+depends: [
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+  "cohttp"
+  "lwt"
+  "uri"
```

Comment:
@avsm any idea why the build fails only for Opam 2.0/Ocaml 4.02 (ocamlfind is unable to find cohttp.lwt which should be available when installing cohttp and lwt together) while the same compiler with opam v1.2 works fine ? It's confusing